index.html

```diff
@@ -32,22 +32,22 @@
         <div class="container" id="home">
             <div class="grid-container">
                 <figure class="item1">
```

Comment:
are u using this class for anything at all? same for item 2, 3 , 4, 5 and 6

Reply:
imageops? Yes, I answerd in the following suggestion.

Reply:
nono item1 , item2, item3, item 4...?